kb/2025-09-30/setup_common_cpu_model_for_vm_live_migration.md

```diff
@@ -0,0 +1,105 @@
+---
+title: Setting up a common CPU model for virtual machine migration
+description: 'How to set up a common CPU model for virtual machine migration'
+slug: setup_common_cpu_model_for_vm_live_migration
+authors:
+  - name: Jack Yu
+    title: Senior Software Engineer
+    url: https://github.com/Yu-Jack
+    image_url: https://github.com/Yu-Jack.png
+tags: [harvester, virtual machine, VM, live migration, policy, strategy, configuration]
+hide_table_of_contents: false
+---
+
+## Problem Description
+
+Although Harvester allows you to perform live migration, it sometimes fails due to mismatched CPU features and models. Let's examine an example.
```

Comment:
```suggestion
For Harvester to successfully migrate a virtual machine from one node to another, the CPU models and features of the origin and target nodes must match.

If the CPU model of a virtual machine isn't specified, Harvester assigns it the `host-model` property so that the it has the CPU model closest to the one that are used on the hosting node.

Let's examine an example.
```